src/components/Navbar/index.jsx

```diff
@@ -97,13 +62,51 @@ const Navbar = ({ region }) => {
                   },
                 }}
               >
-                <Typography variant="h3Bold" sx={{}}>
-                  <Link to={"/signup"} style={{ textDecoration: "none", color: "inherit" }}>
-                    Sign up
-                  </Link>
-                </Typography>
+                <Typography variant="h3">Your region:</Typography>
+                <Select labelId="region-select-label" id="region-select" data-testid="region-select" value={selectedRegion || (regions.length > 0 ? regions[0].public_id : "")} onChange={handleRegionChange}>
```

Comment:
when the value of `selectedRegion` is empty string, it will fallback to get the first index of the data so the UI has value of whatever city on the first index.

sorry I don't know how to review so I just share what I know.

Reply:
I moved that to the following useEffect:

``` useEffect(() => {
    if (regions.length > 0 && !region) {
      setRegion(regions[0]["public_id"]);
    }
  }
  , [regions, region]);```
  
  is this ok?